lib/rucio/db/sqla/migrate_repo/versions/1c45d9730ca6_increase_identity_length.py

```diff
@@ -81,13 +82,13 @@ def downgrade():
         drop_constraint('IDENTITIES_TYPE_CHK', 'identities', type_='check')
         create_check_constraint(constraint_name='IDENTITIES_TYPE_CHK',
                                 table_name='identities',
-                                condition="identity_type in ('X509', 'GSS', 'USERPASS')")
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SAML')")
```

Comment:
in the downgrade, i think you ought to remove SAML here and keep SSH :-)

(same below)

Reply:
Remove SAML means should the account map and identities for SAML be deleted from the table or just remove SAML in the condition above? As of now, SSH keys are deleted from the table so instead of that SAML credentials need to be deleted?

Reply:
the downgrade should bring the database in the same state as it was before. what you are effectively doing here is adding the SAML type in the upgrade, then when you downgrade, instead of removing the SAML you removed the SSH.

Reply:
this applies to the whole migrate script

Reply:
Oh okay, I got it. Sorry Sir.

Reply:
Please create a new migrate script for the adding of `SAML`. This should not be appended to the existing one 😄

Reply:
Sir I was trying to create a new alembic migrate script for SAML now, but it is throwing an error. After talking to hahahannes, it seems that there is a problem with the alembic version on the next branch. So, for now, I have kept this on hold and I will create a new script as soon as the bug on next is fixed. If there is any other workaround then just suggest me and I will do that asap.

Reply:
I opened #2872 for this